utils/helpers.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+/**
+ * Read and parse a YAML file
+ * @param {String} filePath - The path to the YAML file
+ * @returns {Object} An object containing the path and contents of the file
+ */
+const readYamlFile = (filePath) => {
+  const file = fs.readFileSync(filePath);
+  const contents = yaml.loadAll(file);
+  return { path: filePath, contents };
+}
+
+/**
+ * Read and parse a JSON file
+ * @param {String} filePath - The path to the JSON file
+ * @returns {Object} An object containing the path and contents of the file
+ */
+const readJsonFile = (filePath) => {
+  const file = fs.readFileSync(filePath);
+  const contents = JSON.parse(file);
+  return { path: filePath, contents: [ contents ] }; // Return array here to be consistent with the yaml reading
+}
+
+/**
+ * Reads in a JSON or YAML file
+ * @param {String} filePath - The path to the JSON or YAML file
+ * @returns {Object} An object containing the path and contents of the file
+ */
+const readFile = (filePath) => path.extname(filePath) === '.json' ? readJsonFile(filePath) : readYamlFile(filePath);
```

Comment:
im like 50% convinced on this rename, but i think renaming to `readPackFile` might be better, since the usage is about reading in pack files in json or yaml.

Reply:
`readPackFile` makes a lot of sense to me!